src/Tribe/REST/V1/Endpoints/Single_Tag.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+class Tribe__Events__REST__V1__Endpoints__Single_Tag
+	extends Tribe__Events__REST__V1__Endpoints__Term_Single_Base
+	implements Tribe__REST__Endpoints__READ_Endpoint_Interface,
+	Tribe__REST__Endpoints__CREATE_Endpoint_Interface,
+	Tribe__REST__Endpoints__DELETE_Endpoint_Interface,
+	Tribe__REST__Endpoints__UPDATE_Endpoint_Interface,
+	Tribe__Documentation__Swagger__Provider_Interface {
+
+	/**
+	 * Returns the content of the `args` array that should be used to register the endpoint
+	 * with the `register_rest_route` function.
+	 *
+	 * @since TBD
+	 *
+	 * @return array
+	 */
+	public function CREATE_args() {
+		return array(
+			'name'             => array(
+				'required'          => true,
+				'validate_callback' => array( $this->validator, 'is_string' ),
+				'type'              => 'string',
+				'description'       => __( 'The event tag name', 'the-events-calendar' ),
+			),
+			'description'             => array(
```

Comment:
Align these `=>` back with the key, it looks weird when there are too many diff indentations.